login-workflow/src/components/SetPassword/SetPassword.tsx

```diff
@@ -84,14 +88,24 @@ export const SetPassword: React.FC<React.PropsWithChildren<SetPasswordProps>> =
                 inputRef={passwordRef}
                 label={newPasswordLabel}
                 value={passwordInput}
-                onChange={(evt: ChangeEvent<HTMLInputElement>): void => onPassChange(evt.target.value)}
+                error={shouldValidatePassword && !isValidPassword()}
+                sx={TextFieldStyles(theme)}
+                {...passwordTextFieldProps}
```

Comment:
just so I know, why do we have both of these? just for theme aware items? plus additional sx styles?
 sx={TextFieldStyles(theme)}
                {...passwordTextFieldProps}